stat.proto

```diff
@@ -50,6 +50,8 @@ message StatRequest {
     optional int32 end_point_id = 19;
     optional string end_point_name = 20;
     optional StatInfoResponse info_response = 21;
+    optional int32 token_id = 22;
+    optional string token = 23;
```

Comment:
See with @pbougue @vincentlepot @kinnou02 @lrocheWB

We will remove "token_id" field because we already have token information. So we doesn't have to implement this ([YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it))

Thanks